src/solver/effectWConEq.ml

```diff
@@ -87,4 +87,4 @@ module Make =
   end
 
 let _ =
-  Selector.add_solver ("effectWConEq", (module PostSolver.EqIncrSolverFromEqSolver (Make)));
+  Selector.add_solver ("effectWConDemandEq", (module PostSolver.DemandEqIncrSolverFromEqSolver (Make)));
```

Comment:
Do we want to rename the solver here?

Reply:
What exactly does this solver do?

Since this is user facing, the name should relate to what the solver actually does rather than its formal signature. As this one does not really do anything interesting with the demands, it could be confusing to rename.

Reply:
Oh, I had renamed that. Reverted :sweat_smile: